src/test/scala/net/artsy/atomic/AtomicEventStoreSpec.scala

```diff
@@ -76,6 +98,25 @@ class AtomicEventStoreSpec
     testCode(logMaker)
   }
 
+  "TestEvent events" must {
+    "be serializable" in {
+      val serialization = akka.serialization.SerializationExtension(system)
+      def serialize[A <: AnyRef](obj: A) = {
+        val serializer = serialization.findSerializerFor(obj)
+        serializer.toBinary(obj)
+      }
+
+      val event1 = TestEvent1("blah")
+      serialize(event1)
+      serialize(Timestamped(event1, DateTime.now()))
+      serialize(TestEventStore.ConsiderEventFromSender(event1, testActor))
+      serialize(TestEventStore.StoreEvent(Timestamped(event1, DateTime.now())))
+      serialize(TestEventStore.DoNotStoreEvent)
+      serialize(TestEventStore.EventLogAvailable)
+      serialize(TestEventStore.EventLogBusyValidating)
```

Comment:
There is no actual `should` test here. Are these to make sure we don't blow up when these are serialized?